### DISCUSSION TITLE
zIndex not working in Select inside Modal

### DISCUSSION THREAD

**Discussion by vkatsou** [mantinedev/mantine#3774] [Q&A] [answered]
<img width="518" alt="Screenshot 2023-03-14 at 2 28 09 PM" src="https://user-images.githubusercontent.com/11488579/225001055-1fc40995-b264-4317-adb9-5ae49a79189b.png">
Hello all,

I have a problem regarding a select component inside a Modal in **v6.0.1**.
The problem relies on the dropdown that is being hidden under the modal.
I tried raising the zIndex of the Select component to a big number but nothing seems to be working.
I tried playing with the modal zIndex and the Select zIndex but neither that solved the problem.
Here is my block of code:
`
<Modal
            title={'Add Standard Template'}
            onClose={handleOnClose} opened={props.isOpen}>
            <Box>
                <form onSubmit={form.onSubmit(onSubmit)} style={{height: '100%'}}>
                    <Stack h={'100%'} justify={'space-between'}>
                        <Select
                            icon={<IconSearch size={18} stroke={1.3}/>}
                            rightSection
                            placeholder='Search members...'
                            itemComponent={MultiSelectResultsComponent}
                            data={members.map((result: MemberFullDetailsModel) => {
                                return ({
                                    value: result.userId,
                                    id: result.id,
                                    label: result.fullName,
                                    image: result.avatar,
                                });
                            })}
                            zIndex: 10000000000000000
                            clearable
                            searchable
                            {...form.getInputProps('userId')}
                            maxDropdownHeight={200}
                            nothingFound='Nothing matches your search criteria'
                        />
                        <Button loading={status === ProcessStatusEnum.LOADING} type={'submit'}
                                disabled={!form.isDirty() || !form.isValid()}>Submit</Button>
                    </Stack>
                </form>
            </Box>
        </Modal>
`
It would be very kind of you if someone give a feedback about this problem.

**Comment by rtivital** [comment, marked as answer]
Set `<MultiSelect withinPortal />`

**Reply by vkatsou** [reply]
Thanks Vitaly, that did the trick!
